tools/clh-img-check.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+usage="$(basename "$0") [-h] [-f n] -- program to check CLH compatible image
+
+where:
+    -h  show this help text
+    -f  image file location
+    -w directory to be used for temporary files"
```

Comment:
As this working directory is used for temporary files. Is it better to set the "-w" parameter optional? If the user does not care what's in the folder, you can just create a hidden folder somewhere (in local folder or in "/tmp") and remove it after the checking is done.